<launch>

  <!-- If 1, launch file assumes Pathing is started separately -->
  <arg name="included" default="0"/>

  <group unless="$(arg included)">

    <node pkg="cut_mission" type="Pathing.py" name="pathing" output="screen"/>

  </group>

  <node pkg="cut_mission" type="CutPlanner.py" name="cut_planner" output="screen">
    <remap from="/hitch_path" to="/hitch_path"/>
    <param name="file_name" value="$(find cut_mission)/data/scan_file.csv"/>
  </node>

  <node pkg="cut_mission" type="CutBehavior" name="cut_behavior" output="screen">
    <remap from="/hitch_pose" to="/gazebo/tractor_sim_hitch"/>
    <remap from="/twist" to="/state_controller/cmd_behavior_twist"/>
    <remap from="/hitch" to="/state_controller/cmd_behavior_hitch"/>
    <param name="file_name" value="$(find cut_mission)/data/scan_file.csv"/>
  </node>

</launch>
